scripts/postinstall.js

```diff
@@ -0,0 +1,15 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const downloadFile = (async (url, path) => {
+  const res = await fetch(url);
+  const fileStream = fs.createWriteStream(path);
+  await new Promise((resolve, reject) => {
+      res.body.pipe(fileStream);
+      res.body.on("error", reject);
+      fileStream.on("finish", resolve);
+  });
+  fs.chmodSync(path, fs.constants.S_IRWXU);
+});
+
+downloadFile("https://github.com/KaotoIO/kaoto-backend/releases/download/v0.4.0/kaoto-linux-amd64", "./binaries/kaoto-linux-amd64");
```

Comment:
is there any way we can use an environmental variable or something to not have to hardcode the version here?

Reply:
I hope yes. We will need to make it configurable in the context of this other task: #70

This is the only place where the version of kaoto-backend is defined and impact the build. So it doesn't seem needed for now. There is another reference to this version which is part of the readme, not impacting the build. I think there is no built-in support for variables in [GitHub Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github). Also note that the version is different than the kaoto-ui one.